.devcontainer/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM mcr.microsoft.com/devcontainers/cpp:0-ubuntu-20.04
```

Comment:
Why not use `ubuntu:20.04`?

Reply:
Actually, we have a [`Dockerfile`](https://github.com/dartsim/dart-ci-docker-containers/blob/master/v6.13/Dockerfile.ubuntu.focal) that is used for CI and prebuilt image (`dartsim/dart-dev:ubuntu-focal-v6.13`). Maybe we could use it instead without installing any additional packages below? The image is hosted in https://hub.docker.com/repository/docker/dartsim/dart-dev